webplugin/js/app/km-widget-custom-events.js

```diff
@@ -12,24 +12,16 @@ var kmWidgetEvents = {
 
         return (trackingID && trackingID.toString()) || '';
     },
-    sendEventToGoogleAnalytics: function (
-        eventCategory,
-        eventAction,
-        eventLabel,
-        eventValue
-    ) {
+    sendEventToGoogleAnalytics: function (eventObject) {
         try {
             if (
                 kmWidgetEvents.gaTrackingId() &&
-                typeof window.top.ga !== 'undefined'
+                typeof window.top.gtag !== 'undefined'
             ) {
-                window.top.ga('create', kmWidgetEvents.gaTrackingId(), 'auto');
-                window.top.ga('send', {
-                    hitType: 'event',
-                    eventCategory: eventCategory,
-                    eventAction: eventAction,
-                    eventLabel: eventLabel,
-                    eventValue: eventValue,
+                window.top.gtag('event', eventObject.data.eventAction, {
```

Comment:
do we need to update any 3rd party script also?

Reply:
no, basically we are sending events to the user's website ga layer from the widget. there is no direct relation between chat widget and google analytics. Hope this helps. if not let me know